engine/src/main/java/org/camunda/bpm/engine/impl/TaskQueryImpl.java

```diff
@@ -186,6 +187,12 @@ public TaskQueryImpl taskId(String taskId) {
     return this;
   }
 
+  @Override
+  public TaskQueryImpl taskIdIn(String... taskIds) {
+    this.taskIdIn = taskIds;
```

Comment:
We should add a null-check here:
```suggestion
    ensureNotNull("taskIds", (Object[]) taskIds);
    this.taskIdIn = taskIds;
```